TombRaider1996/Component.cs

```diff
@@ -18,6 +18,7 @@ namespace TR1
     internal class Component : AutoSplitComponent
     {
         private Autosplitter _splitter;
+        private LiveSplitState _state;
```

Comment:
VS says this can be made readonly; only thing done is delegation, so it seems correct.
```suggestion
        private readonly LiveSplitState _state;
```

Reply:
Done.